Project.toml

```diff
@@ -10,16 +10,19 @@ Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 NamedDims = "356022a1-0364-5f58-8944-0da4b18d706f"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 
 [compat]
 AxisArrays = "0.4"
 Compat = "3.3"
 Distributions = "0.21, 0.22"
 NamedDims = "0.2"
+Tables = "1"
```

Comment:
Need v1 as there is a bug in v0.2.11 that has been fix now

That means this requires DataFrames v0.20 also

I think that is fine